src/screens/HomeScreen/components/SearchCarousel/SearchCarousel.tsx

```diff
@@ -0,0 +1,61 @@
+import React, {useState} from 'react';
+import {StyleSheet, View} from 'react-native';
+import Dots from 'react-native-dots-pagination';
+import {Carousel} from '../../../../components';
+import {Pallete} from '../../../../theme';
+import {verticalScale, DeviceUtils} from '../../../../utils';
+import {SearchCarouselItem} from './SearchCarouselItem';
+import {SearchCarouselType} from './SearchCarousel.types';
+
+export const SearchCarouselRenderItem = ({index, item}: any) => {
```

Comment:
Type should never be any

Reply:
Yes i have fixed it by creating a look a like type from the carousel's renderItem